core/src/main/java/com/amplifyframework/predictions/result/TextToSpeechResult.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.result;
+
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.core.async.Result;
+
+import java.io.InputStream;
+import java.util.Objects;
+
+/**
+ * The result of the call to synthesize speech from text.
+ */
+public final class TextToSpeechResult implements Result {
```

Comment:
(unrelated to this PR)

Thoughts on renaming `Result` to `Response` (and all subclasses) or something else?   I don't really like `Result` because it might cause some customer confusion with [Kotlin's Result class](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-result/).

Reply:
Re: convos with Tim recently, I'm not sure our `interface Result {}` marker interface is providing us a lot of value. It seems like mostly an architectural vestige at this point. Originally, it might have turned into something like the Swift/Kotlin construct. But we don't really use it for anything important. Could even argue that its harmful, since it _looks_ important, but isn't actually.

My vote is just to `git rm` the thing.

https://github.com/aws-amplify/amplify-android/blob/master/core/src/main/java/com/amplifyframework/core/async/Result.java#L25

Reply:
^ I agree... Result doesn't really serve any purpose.

Meanwhile, calling the actual classes `<purpose>Result` is in alignment with what iOS has, so I don't think I will be changing that to `Response`. Especially if we are removing `Result` class, then there shouldn't be namespace conflict with Kotlin anymore right?

Reply:
https://github.com/aws-amplify/amplify-android/pull/440